ContextualMenuDecorator/tests/ContextualMenuDecorator-specs.js

```diff
@@ -127,4 +127,21 @@ describe('ContextualMenuDecorator Specs', () => {
 
 		expect(contextualMenu).toHaveClass(expected);
 	});
+
+	test('should wrap content in Scroller component if it does not fit inside of popup', () => {
```

Comment:
```suggestion
	test('should wrap content in Scroller component if menuItem length exceeds MAX_VISIBLE_MENU_ITEMS', () => {
```
I think we can make it clearer.